Makefile

```diff
@@ -23,7 +23,22 @@ fmt:
 	goimports -w -l -local github.com/autokitteh/starlark-lsp cmd/ pkg/
 
 install:
-	go install ./cmd/starlark-lsp
+	go install ./cmd/autokitteh-starlark-lsp
 
 builtins:
 	go run ./hack/starlark-builtins.go > pkg/analysis/builtins.py
+
+
+.PHONY: sysroot-pack sysroot-unpack release-dry-run release
+
+PACKAGE_NAME          := github.com/autokitteh/autokitteh-starlark-lsp
+GOLANG_CROSS_VERSION  ?= v1.21  # could use latest, but it's better to specify exact Go version
+
+DOCKER_RUN = docker run --rm -e CGO_ENABLED=1 -v /var/run/docker.sock:/var/run/docker.sock -v `pwd`:/go/src/$(PACKAGE_NAME) -v `pwd`/sysroot:/sysroot -w /go/src/$(PACKAGE_NAME)
+GORELEASER_IMAGE = ghcr.io/goreleaser/goreleaser-cross:${GOLANG_CROSS_VERSION}
```

Comment:
add a space above please 
seems like a one liner but it's two different variables

Reply:
added